### PR TITLE
Add contributing.md and pull_request_template.md files

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

**Review by eemeli — APPROVED** [review]
The `spec/` links in contributing.md should be fully qualified.

**Comment by Thom1729** [comment]
`contributing.md` is fine. I'm not sure about the PR template.

File: .github/contributing.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+For now we ask that you **don't submit RFCs** or comment on the ones that we
+attempted to start earlier.
+
+
+## Using the Build System
+
+This repository has a build system for turning the spec source files into HTML
+pages.
+Even though the project is built with a lot of open source framework, the only
```
Comment (perlpunk) [inline .github/contributing.md:35]: "a lot of open source framework**s**"?

Comment (ingydotnet) [reply .github/contributing.md]: Good catch. Thanks. Fixed in 12553eb

**Pull request merged**